libs/application-generic/src/usecases/create-workflow/create-workflow.usecase.ts

```diff
@@ -114,8 +116,9 @@ export class CreateWorkflow {
 
     try {
       if (
-        process.env.NOVU_ENTERPRISE === 'true' ||
-        process.env.CI_EE_TEST === 'true'
+        (process.env.NOVU_ENTERPRISE === 'true' ||
+          process.env.CI_EE_TEST === 'true') &&
+        storedWorkflow.origin === WorkflowOriginEnum.NOVU_CLOUD_V1
```

Comment:
only run the translation analytics code in v1

Reply:
Ideally, we should not load the translations during the message sent in the `SendMessageBase` usecase, but I haven't added any logic there because there is no workflow information at that stage, like the `NotificationStepEntity` is missing the `workflow` info and particularly the `origin` field. 
To do that, it might require more work.